pubobot/bot.py

```diff
@@ -1291,9 +1291,16 @@ def lastgame(self, member, args, index):
                 players = "{0}\n{1}".format(l[4], l[5])
             else:
                 players = l[3]
+
+            escaped_players = (
+                players.replace("`", r"\`")
+                .replace("_", r"\_")
+                .replace("*", r"\*")
+                .replace("#", r"\#")
+            )
```

Comment:
How about implement an `escape_markdown()` function and do this
```python
import re

def escape_markdown(s):
  return re.replace(r"([_*#`])", r"\\\1", s)
```
Then just use that wherever we need to escape characters.